src/index.tsx

```diff
@@ -167,39 +148,56 @@ function ResponsiveDrawer() {
                         </Toolbar>
                     </AppBar>
                     <nav className={classes.drawer}>
-                        <Hidden smUp>
-                            <Drawer
-                                variant="temporary"
-                                anchor={theme.direction === 'rtl' ? 'right' : 'left'}
-                                open={mobileOpen}
-                                onClose={handleDrawerToggle}
-                                classes={{ paper: classes.drawerPaper }}
-                                ModalProps={{ keepMounted: true }}>
-                                {drawer}
-                            </Drawer>
-                        </Hidden>
                         <Hidden xsDown>
                             <Drawer classes={{ paper: classes.drawerPaper }} variant="permanent" open>
                                 {drawer}
                             </Drawer>
                         </Hidden>
                     </nav>
-                    <main className={classes.content}>
+                    <main className={classes.content} style={{ display: 'relative' }}>
                         <div className={classes.toolbar} />
-                        {OptionsPageRouters}
+                        <div style={{ display: 'absolute' }}>{OptionsPageRouters}</div>
+                        <Hidden smUp>
+                            <div className={classes.bottomNavigationMargin} />
+                            <BottomNavigation classes={{ root: classes.bottomNavigationRoot }}>
+                                {Links1st.props.children}
+                                {Links2nd.props.children}
+                            </BottomNavigation>
+                        </Hidden>
                     </main>
                 </div>
             </Router>
         </ThemeProvider>
     )
 }
-function LinkItem(props: { to: string; icon: React.ReactElement; title: string }) {
+function MobileBackButton() {
+    if (!(webpackEnv.firefoxVariant === 'GeckoView' || webpackEnv.target === 'WKWebview')) return null
     return (
+        <MuiLink color="textPrimary" onClick={window.close}>
```

Comment:
Is that a close button? Use a "close" button if your back button is really intended to close the popup, or use `window.history.back` for back. See the semantics at [material.io](https://material.io/design/navigation/understanding-navigation.html).

The spacing on that button looks awkward. Specifically:
* There should be more space to the right of the arrow. The placement should be identical to the `menuButton` element in index.ts.
* The baseline of the logo looks off. Should be fixed by applying the `menuButton` margin: as long as it's not directly next to the IconButton it should be fine.

Reply:
this button only appears on Android or iOS standalone app. there is no "tabs" equivalent in those environment so clicking on the back button behaves like "back" from the settings page to the Facebook.com